tensorflow_lite_support/python/test/task/vision/image_searcher_test.py

```diff
@@ -35,6 +35,28 @@
 _MOBILENET_EMBEDDER_MODEL = 'mobilenet_v3_small_100_224_embedder.tflite'
 _MOBILENET_SEARCHER_MODEL = 'mobilenet_v3_small_100_224_searcher.tflite'
 _MOBILENET_INDEX = 'searcher_index.ldb'
+_EXPECTED_MOBILENET_DEFAULT_OPTIONS_SEARCH_RESULT = """
```

Comment:
Please create a SearchResult object to define the expected results instead of using protobuf here.

Reply:
Fixed it